db/migrate/20220705132207_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
```

Comment:
Please add `author_id` instead of `user_id` to comments table as it can be seen in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Please add project descriptions and instructions to set up the project locally to README.